src/backtrace.c

```diff
@@ -99,6 +101,21 @@ static int backtrace_thread(unw_accessors_t *accessors, void *arg)
     return rc;
 }
 
+void print_thread_heading(const int *index, const int *tids,
+        const char *states, int i)
```

Comment:
Maybe use `const char states[]` to make it clear that it's not C-string, but array of chars?

Reply:
Done: https://github.com/tbricks/tbstack/pull/14/commits/54550c0a90f40441e5e5fa1ce9cfdaff4533114e